### PR TITLE
Allowing errors from Jimp to be passed back to the user

### PR DESCRIPTION
**Pull request by silverlight513**
Was getting unhandled promise errors from jimp.read and getSize methods. 

Added a few catch statements so we can pass errors back to the processImage callback.

### CODE REVIEW

File: src/components/ProcessImage.js [hunk 1]
```diff
@@ -114,26 +119,45 @@ class ProcessImage extends Component {
...
-      });
+      const result = await worker
+        .process({
+          props: filterPropsToListen(props),
+          image: props.image,
+          customCdn: props.customCdn
+        })
+        .catch(err => ({ err }));
+
+      if (typeof result.src === 'undefined' && result.err) {
```
Comment (silverlight513) [inline src/components/ProcessImage.js]: I created this if statement to break out of an infinite loop. componentDidUpdate calls this method and the image would try to be loaded again causing another error.

**Comment by silverlight513** [comment]
@Sunshine168 got another PR for you with some added error handling

**Comment by Sunshine168** [comment]
Thanks and I will check it later and seem your commit 「Merge remote-tracking branch 'source/master'」 is not correct  ?  It should be just rebase to your local branch for remote master .

**Comment by silverlight513** [comment]
That's an automatic commit name from pulling in latest code. I had conflicts on my branch and therefore had to do a merge

**Comment by Sunshine168** [comment]
Thanks !

**Pull request merged**